stdlib/auth/src/main/ballerina/src/auth/outbound_basic_auth_provider.bal

```diff
@@ -46,7 +49,8 @@ public type OutboundBasicAuthProvider object {
                     return authToken;
                 }
             }
-            return prepareError("Failed to generate basic auth token since credential config is not defined and auth token is not defined in the authentication context at invocation context.");
+            return prepareError("Failed to generate basic auth token since credential config is not defined and auth " +
+                            "token is not defined in the authentication context at invocation context.");
```

Comment:
This was done intentionally because when we print the error logs after this change, it will print with new line character. So, shall we revert this change? This need to be properly fixed in log-api.
```suggestion
            return prepareError("Failed to generate basic auth token since credential config is not defined and auth token is not defined in the authentication context at invocation context.");
```